drivers/spi/spi_xec_qmspi_ldma.c

```diff
@@ -1105,6 +1118,13 @@ static int qmspi_xec_init(const struct device *dev)
 		return ret;
 	}
 
+	ret = clock_control_get_rate(DEVICE_DT_GET(MCHP_XEC_CLOCK_CONTROL_NODE),
```

Comment:
Is there a missing error path here if device_is_ready() is false for this device?

Reply:
will add device_is_ready() check